examples/Example.java

```diff
@@ -18,19 +18,15 @@ public static void main(String[] args) throws IOException {
 
     Request request = new Request();
     request.baseUri = "api.sendgrid.com";
-    Map<String,String> requestHeaders = new HashMap<String, String>();
-    requestHeaders.put("Authorization", "Bearer " + System.getenv("SENDGRID_API_KEY"));
-    request.headers = requestHeaders;
+    request.getHeaders().put("Authorization", "Bearer " + System.getenv("SENDGRID_API_KEY"));
```

Comment:
Can we use a setter here instead?

e.g. `request.addHeader("Authorization", "Bearer " + System.getenv("SENDGRID_API_KEY"));`